src/registrar/assets/sass/_theme/_accordions.scss

```diff
@@ -31,3 +31,17 @@
         margin-top: 0 !important;
     }
 }
+
+.usa-accordion--select .usa-accordion__content {
+    top: 33.88px;
```

Comment:
We have a quirk. The accordion body which is positioned absolute needs, in this case, to align with the top of the accordion button. The accordion button's element height when padding, line height, etc. are calculated is 33.88px. 

There _might_ be a way to avoid this but it would entail changing the positions and displays of most of the accordion elements. I don't think this is worth it.